examples/surface_mesh_airplane_v2_from_inputs.py

```diff
@@ -0,0 +1,29 @@
+import flow360 as fl
+from flow360.examples import Airplane
+
+os.environ["FLOW360_BETA_FEATURES"] = "1"
+
+params = fl.SurfaceMeshingParams(
+    version="v2",
+    max_edge_length=0.16,
+    edges={
```

Comment:
I don't think the v2 mesher supports edges or faces, let's just keep max_edge_length

Reply:
fixed